src/RuleSet/Sets/PhpCsFixerSet.php

```diff
@@ -110,6 +110,32 @@ public function getRules(): array
             'php_unit_test_class_requires_covers' => true,
             'phpdoc_add_missing_param_annotation' => true,
             'phpdoc_no_empty_return' => true,
+            'phpdoc_order' => [
+                'order' => [
```

Comment:
Can we have the order extracted to constant with some meaningful (no idea what) name?

Reply:
See the whole `PhpCsFixerSet` class, other PhpCsFixer ruleset configs do not have constants neither, therefore I prefer to keep this PR as is and address this for all "larger rulesets" constants at once later.

Reply:
I agree that using constant here is superfluous, as it does not bring value (no re-usage anywhere) but at the same time requires additional navigation jumps to grasp the config.